src/components/Editor/Resources/ResourceListItem.vue

```diff
@@ -40,22 +40,45 @@
 					@click="removeResource">
 					{{ $t('calendar', 'Remove resource') }}
 				</ActionButton>
-			</Actions>
+				<ActionSeparator />
+				<ActionCaption
+					v-if="seatingCapacity"
+					:title="seatingCapacity" />
+				<ActionCaption
+					v-if="roomType"
+					:title="roomType" />
+				<ActionCaption
+					:v-if="hasProjector"
+					:title="$t('calendar', 'Projector')" />
```

Comment:
Nitpick: should we maybe word this as *Has a projector* or *With projector*? The text lacks a bit of context otherwise

Reply:
I changed the wording of projector and whiteboard entries.